### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by nigelteosw**

### CODE REVIEW

**Review by Jnjy — APPROVED** [review]
some formatting issue, let the dailou @Bisceto review

File: docs/UserGuide.md [hunk 1]
```diff
@@ -71,12 +71,19 @@ Format: `help`
...
-Format: `placeholder`
+Format: `add`
 
 Examples:
-* `placeholder`
+* `add Spinach`
+* `add Meiji Milk`
+* `add Awfully Chocolate`
+
+Command outcome:
```
Comment (Jnjy) [inline docs/UserGuide.md:83]: the format of the example differs from the current format that is currently in the user guide now, may need to change the format

**Review by Jnjy — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -71,12 +71,19 @@ Format: `help`
...
+```shell
+  You have successfully added Spinach into your fridge.
+```
```
Comment (Jnjy) [inline docs/UserGuide.md:86]: example format is different from the current user guide, need to change the formatting

**Review by Jnjy — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (nigelteosw) [reply docs/UserGuide.md]: Changes have been made to follow the other formats

**Review by Bisceto — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -79,12 +79,20 @@ Tag food item - tag <Index> /with <Tag Name>
...
+Add food items into your fridge.
 
-Format: `placeholder`
+Format: `add`
 
 Examples:
-* `placeholder`
+* `add Spinach`
+* `add Meiji Milk`
+* `add Awfully Chocolate`
```
Comment (Jnjy) [inline docs/UserGuide.md:89]: I think can remove this since there is already an example section below, other than that should it looks good

**Review by Jnjy — APPROVED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (nigelteosw) [reply docs/UserGuide.md]: removed items

**Pull request merged**